deploy_pio-dist.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+rm -rf target
+
+mkdir target
+mkdir target/pio-dist
+
+cp pio-dist/* target/pio-dist/
+
+cd target
+tar -czf pio-dist.tar.gz pio-dist
+aws s3 cp pio-dist.tar.gz s3://prediction-io-dist/mobile-prediction-io/PROD/recommender/pio-dist.tar.gz
```

Comment:
If pio-dist is used by recommender, eventserver and the training server, it should be in s3://prediction-io-dist/mobile-prediction-io/PROD/common or s3://prediction-io-dist/mobile-prediction-io/PROD/pio-dist